pkg/destinations/redshift/redshift.go

```diff
@@ -0,0 +1,41 @@
+package redshift
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/lib/pq"
+	"github.com/scratchdata/scratchdata/util"
+)
+
+type RedshiftServer struct {
```

Comment:
Each Redshift destination will have a separate credential for the S3 bucket it uses to upload. In addition to the DB config options, we need to include S3 credentials. 

In `Step 2` of the ticket, you can see the additional fields that we need for S3.